components/k8s-model-server/docker/Makefile

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 VERSION=1.0
+GPU_VERSION=1.4
```

Comment:
this is gpu version or tensorflow version or tensorflow-gpu version?

Reply:
TensorFlow Version.